### PR TITLE
tweaks to project list to support involuntary conversions

### PR DESCRIPTION
**Pull request by jordan-beard**
image

### CODE REVIEW

File: Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml [hunk 1]
```diff
@@ -2,151 +2,158 @@
...
+                                <tr class="govuk-table__row" data-cy="select-projectlist-filter-row">
+                                    <td class="govuk-table__cell">
+                                        <h2 class="govuk-caption-l govuk-!-margin-bottom-0 govuk-!-margin-top-1">
+                                            <strong>
+                                                <a id="@("school-name-" + index)" class="govuk-link" asp-page="@Links.TaskList.Index.Page" asp-route-id="@project.Id">@project.SchoolName</a>
+                                            </strong>
+                                            <span id="@("urn-" + index)">URN @project.SchoolURN</span>
+                                        </h2>
+                                        <p class="govuk-!-margin-top-3">
+                                            <div data-cy="route">Route: @(string.IsNullOrWhiteSpace(project.ApplicationReceivedDate) ? "Involuntary conversion" : "Voluntary conversion")</div>
```
Comment (dneed-nimble) [inline Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml:91]: Have we tested updating an involuntary project yet? To check the lack of an application doesn't throw a null ref at any point? (probably outside of this PR)

Comment (jordan-beard) [reply Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml:91]: Good question, will check.

Comment (jordan-beard) [reply Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml:91]: Updating seems to work. And generating a template also works. I expect parts of the system will not but I don't want to go through each form to make sure it works with an involuntary project. @rdogra-nimble any thoughts on this? Maybe there's a way of automating this check.

Comment (rdogra-nimble) [reply Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml:91]: @jordan-beard @dneed-nimble School App. form is not displaying in Involuntary convers projects yet in Dev. env. 
Updating seems works fine! Generating a template as well. Yes, we can cover in automation :)

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**